wakaama.cmake

```diff
@@ -323,6 +323,25 @@ function(target_sources_shared target)
     target_include_directories(${target} PUBLIC ${WAKAAMA_EXAMPLE_SHARED_DIRECTORY})
 endfunction()
 
+# Static library that users of Wakaama can link against
```

Comment:
I do not understand why having a static library is a good thing to have in the first place. AFAIR, the Wakaama way of doing things is "here are the sources, and there are 100x configuration parameters, pick yours and build it".

Reply:
Wakaama became difficult to build. Where to set which defines and compiler flags, what files to include into the build, which include directories to use... That's after all, that's why we have build systems and software libraries. You will still need to pick your configuration parameters, but you don't need to care about the rest.

Reply:
It's almost impossible to modularize the project without the use of a capable build system. Currently, the project is very monolithic and hard to maintain.

Reply:
I think this comment would be good to have in the commit message and/or in wakaama.cmake.